schema/README.md

```diff
@@ -0,0 +1,3 @@
+# Schema
+
+Rocket's database definition is in `tables.sql`. When adding new fields/modifying Rocket's database, add a migration to `./migrations/`.
```

Comment:
I don't think this is worth it's own README. Can we just leave it in the main README?

Reply:
Sure, just felt bad that all the other folders got docs and not this guy